packages/samples/vuejs/README.md

```diff
@@ -1,19 +1,157 @@
-# vuejs
+# Vue.js Search Page
 
 ## Project setup
-```
+
+```sh
 npm install
 ```
 
 ### Compiles and hot-reloads for development
-```
+
+```sh
 npm run serve
 ```
 
 ### Compiles and minifies for production
-```
+
+```sh
 npm run build
 ```
 
 ### Customize configuration
+
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+&nbsp;
+
+## Configuring `@coveo/atomic`
+&nbsp;
+
+### Add Vue configuration to help it handle atomic's components
+
+```js
+// vue.config.js
+{
+  ...,
+  chainWebpack: (config) => {
```

Comment:
Is there a way we could export a plugin which does this configuration so our clients don't have to do it themselves? I feel like that would also be easier to maintain if we need to change it in the future.